docs/source/install.md

```diff
@@ -24,24 +23,34 @@ I/O support.
 
 ### PyPI
 
-Ready-to-use (compiled) distributions are not yet available on PyPI because it
-depends on including compiled binary dependencies. We are planning to release
-compiled distributions on PyPI for Linux and MacOS relatively soon.
+This package is available on [PyPI](https://pypi.org/project/pyogrio/) for Linux,
+MacOS, and Windows.
 
-We are unlikely to release Windows packages on PyPI in the near future due to
-the complexity of packaging binary packages for Windows. If you are interested
-in helping us develop a packaging pipeline for Windows, please reach out!
+```bash
+pip install pyogrio --pre
+```
+
+(Once 0.4.0 is officially released, you can omit the `--pre`)
```

Comment:
Can we actually try to remove this and `--pre` before the release so the official docs for stable 0.4.0 has only `pip install pyogrio`?